script.js

```diff
@@ -0,0 +1,20 @@
+//JavaScript for Kids
+var secondsInAMinute = 60 ; 
+var minutesInAHour = 60 ;
+var secondsInAHour =  secondsInAMinute * minutesInAHour;
+console.log (secondsInAHour)
```

Comment:
we do not need space before opening parentheses of function calls ie. `console.log(secondsInAHour)`